live-examples/js-examples/globalprops-parsefloat.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">function circumference(r) {
+  return parseFloat(r) * 2.0 * Math.PI;
+}
+
+console.log(circumference(4.567));
+// expected output: 28.695307297889173
+
+console.log(circumference("4.567abcdefgh"));
+// expected output: 28.695307297889173
+
+console.log(circumference("abcdefgh"));
+//expected output: NaN
+</code>
+</pre>
```

Comment:
really nice example :)

Reply:
Thanks :) I tried to simplify after your previous review - hopefully this works.